### PR TITLE
Improve buttons  layout

### PR DESCRIPTION
**Pull request by shadabkhan16**
## Description
Please include a summary of the changes and the purpose of this PR.

- What does this PR do?
- Why is it needed?

## Screenshots
Screenshot 2024-12-07 at 12 45 43 PM

### Desktop:
[Add screenshot here]

### Mobile:
[Add screenshot here]

## Approach
Briefly explain the approach taken to implement the changes. 

- How did you solve the problem?
- Any particular design patterns or optimizations?

## Issue Number (if any)
Link to the issue this PR addresses, if applicable:

Fixes #[issue_number]

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/game_frontend/src/pages/Dashboard/InviteFriends/index.tsx [hunk 1]
```diff
@@ -75,19 +75,19 @@ const InviteFriends: React.FC = () => {
         </div>
 
         <div className="referral-button-container">
-          <Button>
+          <Button className="text-sm md:text-2xl lg:text-4xl">
```
Comment (ArjunQBTech) [inline src/game_frontend/src/pages/Dashboard/InviteFriends/index.tsx:78]: Don't use tailwind css directly

File: src/game_frontend/src/pages/Dashboard/InviteFriends/index.tsx [hunk 2]
```diff
@@ -75,19 +75,19 @@ const InviteFriends: React.FC = () => {
...
             BECOME A REFERRAL
           </Button>
         </div>
 
         <FriendsList />
 
         <div className="bottom-buttons-container">
-          <Button className="invite-button">
+          <Button className="invite-button text-sm md:text-2xl lg:text-4xl ">
```
Comment (ArjunQBTech) [inline src/game_frontend/src/pages/Dashboard/InviteFriends/index.tsx:86]: Don't use Tailwind css directly

File: src/game_frontend/src/pages/Dashboard/InviteFriends/index.tsx [hunk 3]
```diff
@@ -75,19 +75,19 @@ const InviteFriends: React.FC = () => {
...
             INVITE A FRIEND
           </Button>
 
-          <Button>
+          <Button  size={'xs'}>
```
Comment (ArjunQBTech) [inline src/game_frontend/src/pages/Dashboard/InviteFriends/index.tsx:90]: Button does not accepts `size='xs'`

**Review by ArjunQBTech — CHANGES_REQUESTED** [review]

**Comment by shadabkhan16** [comment]
Screenshot 2024-12-09 at 6 12 09 PM

**Review by ArjunQBTech — APPROVED** [review]

**Pull request merged**